docs/how-to-use-gvfs.md

```diff
@@ -0,0 +1,225 @@
+# you should put the properties above into this file
+vi ${HADOOP_HOME}/etc/hadoop/core-site.xml
+
+# 2. Place the gvfs runtime jar into your Hadoop environment
+cp gravitino-filesystem-hadoop3-runtime-{version}.jar ${HADOOP_HOME}/share/hadoop/common/lib/
+
+# 3. Complete the Kerberos authentication of the Hadoop environment (if necessary).
+# You need to ensure that the Kerberos has permission to operate the HDFS directory.
+kinit -kt your_kerbers.keytab your_kerberos@xxx.com
+
+# 4. Try to list the fileset
+./${HADOOP_HOME}/bin/hadoop dfs -ls gvfs://fileset/test_catalog/test_schema/test_fileset_1
+```
+
+### Use GVFS via Java code
+
+You can also operate the files or directories managed by fileset through Java code.
+Please be sure that your code is running in the correct Hadoop environment, and the environment
+has the `gravitino-filesystem-hadoop3-runtime-{version}.jar` dependency.
+
+For example:
+
+```java
+Configuration conf = new Configuration();
+conf.set("fs.AbstractFileSystem.gvfs.impl","com.datastrato.gravitino.filesystem.hadoop3.Gvfs");
+conf.set("fs.gvfs.impl","com.datastrato.gravitino.filesystem.hadoop3.GravitinoVirtualFileSystem");
+conf.set("fs.gravitino.server.uri","http://localhost:8090");
+conf.set("fs.gravitino.client.metalake","test_metalake");
+Path filesetPath = new Path("gvfs://fileset/test_catalog/test_schema/test_fileset_1");
+FileSystem fs = filesetPath.getFileSystem(conf);
+fs.getFileStatus(filesetPath);
+```
+
+### Use GVFS with Apache Spark
+
+1. Add the gvfs runtime jar to the Spark environment.
+
+    You can use `--packages` or `--jars` in the Spark submit shell to include the Gravitino Virtual
+    File System runtime jar, like this:
+
+    ```shell
+    ./${SPARK_HOME}/bin/spark-submit --packages com.datastrato.gravitino:filesystem-hadoop3-runtime:${version}
+    ```
+
+    If you want to include Gravitino Virtual File System runtime jar in your Spark installation,
+    make sure to add it to the `${SPARK_HOME}/jars/` folder.
+
+2. Configure the Hadoop configuration when submit the job
+
+    Then, you can configure the Hadoop configuration in the submit shell command like this:
+
+    ```shell
+    --conf spark.hadoop.fs.AbstractFileSystem.gvfs.impl=com.datastrato.gravitino.filesystem.hadoop3.Gvfs
+    --conf spark.hadoop.fs.gvfs.impl=com.datastrato.gravitino.filesystem.hadoop3.GravitinoVirtualFileSystem
+    --conf spark.hadoop.fs.gravitino.server.uri=${your_gravitino_server_uri}
+    --conf spark.hadoop.fs.gravitino.client.metalake=${your_gravitino_metalake}
+    ```
+
+3. Operate the fileset storage in your code
+
+    Finally, you can operate the fileset storage in your Spark program:
+
+    ```scala
+    // Scala
+    val spark = SparkSession.builder()
+          .appName("Gvfs Example")
+          .getOrCreate()
+
+    val rdd = spark.sparkContext.textFile("gvfs://fileset/test_catalog/test_schema/test_fileset_1")
+
+    rdd.foreach(println)
+    ```
+
+
+### Use GVFS with Tensorflow.
```

Comment:
@jerryshao I have added docs about TensorFlow. Please take a look and improve it if needed.